overrides.nix

```diff
@@ -264,4 +270,10 @@ in skipOverrides {
 
   };
 
+  "tox" = self: old: {
+    buildInputs = old.buildInputs ++ [self."setuptools-scm"];
```

Comment:
because the version of tox will not be detected properly otherwise.  If we don't build the package with setuptools_scm, the tox version will be "0.0.0" which will cause all sorts of prolems if other applications require a specific version of that package

Reply:
oh i didn't know about that. that sucks. i guess we would also have to set `use_scm_version` to False to completely ignore setuptools_scm, but I see that this will only create more problems.